Dashboard/Views/StatusView.axaml

```diff
@@ -1,8 +1,246 @@
+                                        <ColumnDefinition Width="Auto" />
+                                    </Grid.ColumnDefinitions>
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition Height="Auto" />
+                                        <RowDefinition Height="Auto" />
+                                    </Grid.RowDefinitions>
+                                    <Slider Grid.Row="0" Minimum="-100" Maximum="100" Value="17" Width="800" IsEnabled="False" Grid.Column="0" Margin="0,0,20,0"/>
+                                    <TextBlock Text="Target" VerticalAlignment="Center" Grid.Column="0" Grid.Row="1"/>
+                                    <TextBlock Grid.Row="0" Text="17º" FontSize="22" VerticalAlignment="Center" Background="White" Padding="20,0" TextAlignment="Center" Grid.Column="1"/>
+                                </Grid>
+                                <Grid HorizontalAlignment="Left" VerticalAlignment="Top" Margin="0,0,0,0">
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition Height="Auto" /> <!-- Speed Actual Row -->
+                                        <RowDefinition Height="Auto" /> <!-- Speed Target Row -->
+                                    </Grid.RowDefinitions>
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="Auto" />
+                                        <ColumnDefinition Width="Auto" />
+                                        <ColumnDefinition Width="Auto" />
+                                    </Grid.ColumnDefinitions>
+
+                                    <!-- Speed Actual Text -->
+                                    <StackPanel Grid.Row="0" Grid.Column="0" Orientation="Horizontal" VerticalAlignment="Center" Margin="0,10,10,10">
+                                        <TextBlock Text="Speed - Actual:" FontSize="24" VerticalAlignment="Center" />
+                                        <TextBlock Text="20 km/h" FontSize="24" Padding="20,0" Background="White" Margin="10,10,10,10" VerticalAlignment="Center"/>
+                                    </StackPanel>
+                                    <StackPanel Grid.Row="0" Grid.Column="0" Orientation="Horizontal" VerticalAlignment="Center" Margin="0,90,10,10">
+                                        <TextBlock Margin="0,0,10,10" Text="Speed - Target:" FontSize="24" VerticalAlignment="Center"/>
+                                        <TextBlock Text="24 km/h" FontSize="24" Padding="20,0" Background="White" Margin="10,10,10,10" VerticalAlignment="Center"/>
+                                    </StackPanel>
+
+                                    <!-- Speedometer Canvas -->
+                                    <StackPanel Grid.Row="0" Grid.Column="1">
+                                        <Canvas Width="300" Height="300">
```

Comment:
The width of these gauges are a bit messed up, they are larger than 300 wide so layout wise these are considered 300 wide, but the canvas is larger and will draw bigger than it actually is

You can check this by going into the dev tools, launch the app and press f12

Reply:
To be honest, I am not sure how to fix this 100%. If I remove the Width & Height values, it messes with the layout of the RES Icon. I just updated them to Width="350" Height="250" for now, can you please double check after my next commit if that's better?

Reply:
I think it is better now, still not ideal. We can revisit this in the future also, not the end of the world